src/evm-config.js

```diff
@@ -214,32 +208,11 @@ function sanitizeConfig(name, config, overwrite = false) {
     changes.push(`fixed invalid property ${color.config('reclient: none')}`);
   }
 
-  if (config.reclient !== 'none' && config.goma !== 'none') {
-    config.goma = 'none';
-    changes.push(`disabled ${color.config('goma')} as ${color.config('reclient')} is enabled`);
-  }
-
   if (!('preserveXcode' in config)) {
     config.preserveXcode = 5;
     changes.push(`defined ${color.config('preserveXcode')} to default value of 5`);
   }
 
-  const gomaGnArg = `import("${goma.gnFilePath}")`;
-  const hasGomaImport = !(
-    !config.gen ||
-    !config.gen.args ||
-    !config.gen.args.find(arg => arg.includes(goma.gnFilePath))
-  );
-  if (config.goma !== 'none' && !hasGomaImport) {
-    config.gen = config.gen || {};
-    config.gen.args = config.gen.args || [];
-    config.gen.args.push(gomaGnArg);
-    changes.push(`added ${color.cmd(gomaGnArg)} needed by goma`);
-  } else if (config.goma === 'none' && hasGomaImport) {
-    config.gen.args = config.gen.args.filter(arg => !arg.includes(goma.gnFilePath));
-    changes.push(`removed gn arg ${color.cmd(gomaGnArg)} as goma is disabled`);
-  }
-
   const remoteExecGnArg = 'use_remoteexec = true';
```

Comment:
```suggestion
  if (config.goma) {
    delete config.goma;
    changes.push(`removed deprecated ${color.config('goma')} property`);
  }

  if (config.gomaSource) {
    delete config.gomaSource;
    changes.push(`removed deprecated ${color.config('gomaSource')} property`);
  }

  const remoteExecGnArg = 'use_remoteexec = true';
```

We should clean up configs which still have these keys, otherwise they will hit a validation error:

```
ERROR Config file testing had the following validation errors:
[
  {
    "instancePath": "",
    "schemaPath": "#/additionalProperties",
    "keyword": "additionalProperties",
    "params": {
      "additionalProperty": "goma"
    },
    "message": "must NOT have additional properties"
  }
]
```